sdk-rs/src/dlob.rs

```diff
@@ -110,6 +116,78 @@ impl DLOBClient {
             }
         });
 
+        RxStream(rx)
+    }
+
+    /// Subscribe to an orderbook via WebSocket.
+    pub async fn subscribe_ws(&self, market: MarketId) -> L2OrderbookStream {
+        // This unwrap should never panic
+        let ws_url = crate::utils::http_to_ws(&self.url).unwrap();
+        let (mut ws_stream, _) = connect_async(ws_url).await.unwrap();
```

Comment:
return `SdkResult<L2OrderBookStream>` helpful to surface all the errors e.g.
```suggestion
        let (mut ws_stream, _) = connect_async(ws_url).await?;
```